propel/internal/utils/utils.go

```diff
@@ -5,13 +5,31 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/meta"
 
+	client "github.com/propeldata/terraform-provider-propel/propel_client"
 	"github.com/propeldata/terraform-provider-propel/version"
 )
 
+var (
+	dataSourceTypeMap = map[client.DataSourceType]string{
+		client.DataSourceTypeSnowflake: "Snowflake",
+		client.DataSourceTypeS3:        "S3",
+		client.DataSourceTypeHttp:      "Http",
+		client.DataSourceTypeWebhook:   "Webhook",
```

Comment:
Should we add the others, too?

Reply:
no in this PR, I'll add those in the PR that allows to create those kind of Data Sources